dashboards/amazon-inspector/amazon-inspector.json

```diff
@@ -0,0 +1,271 @@
+{
+    "name": "Amazon Inspector",
+    "description": null,
+    "pages": [
+      {
+        "name": "Amazon Inspector",
+        "description": null,
+        "widgets": [
+          {
+            "title": "Total assessment runs",
+            "layout": {
+              "column": 1,
+              "row": 1,
+              "width": 4,
+              "height": 3
+            },
+            "visualization": {
+              "id": "viz.billboard"
+            },
+            "rawConfiguration": {
+              "facet": {
+                "showOtherSeries": false
+              },
+              "nrqlQueries": [
+                {
+                  "accountId": 0,
+                  "query": "FROM Metric SELECT sum(aws.inspector.TotalAssessmentRuns) as 'TotalAssessmentRuns' "
+                }
+              ],
+              "platformOptions": {
+                "ignoreTimeRange": false
+              }
+            }
+          },
+          {
+            "title": "Total findings",
+            "layout": {
+              "column": 5,
+              "row": 1,
+              "width": 4,
+              "height": 3
+            },
+            "visualization": {
+              "id": "viz.billboard"
+            },
+            "rawConfiguration": {
+              "facet": {
+                "showOtherSeries": false
+              },
+              "nrqlQueries": [
+                {
+                  "accountId": 0,
+                  "query": "FROM Metric SELECT sum(aws.inspector.TotalFindings) AS 'TotalFindings'"
+                }
+              ],
+              "platformOptions": {
+                "ignoreTimeRange": false
+              }
+            }
+          },
+          {
+            "title": "",
+            "layout": {
+              "column": 9,
+              "row": 1,
+              "width": 4,
+              "height": 3
+            },
+            "visualization": {
+              "id": "viz.markdown"
+            },
+            "rawConfiguration": {
+              "text": "# Amazon Inspector\nAmazon Inspector is a vulnerability management service that continuously scans your AWS workloads for software vulnerabilities and unintended network exposure. Amazon Inspector automatically discovers and scans running Amazon EC2 instances, container images in Amazon Elastic Container Registry (Amazon ECR), and AWS Lambda functions for known software vulnerabilities and unintended network exposure."
+            }
+          },
+          {
+            "title": "Total findings by AssessmentTargetName",
```

Comment:
```suggestion
            "title": "Total findings by assessment target name",
```

Reply:
@mdumpati As discussed we will continue the same way. Please take ahead this PR for approval if there are no more comments